lib/Serialization/ModuleFormat.h

```diff
@@ -56,7 +56,7 @@ const uint16_t SWIFTMODULE_VERSION_MAJOR = 0;
 /// describe what change you made. The content of this comment isn't important;
 /// it just ensures a conflict if two people change the module format.
 /// Don't worry about adhering to the 80-column limit for this line.
-const uint16_t SWIFTMODULE_VERSION_MINOR = 621; // protocol associated type list
+const uint16_t SWIFTMODULE_VERSION_MINOR = 623; // new BodyKind for AbstractFunctionDecl
```

Comment:
After a rebase with `main` before merging, we'll need to increment `main`'s version again, since it got outdated.

Reply:
Right, done -- just incremented by +1 again, that's correct right?

Reply:
yup that's all you need to do. +1 on whatever is on `main`.